### DISCUSSION TITLE
Menu issues with self-compiled Gittyup

### DISCUSSION THREAD

**Discussion by cova-fe** [Murmele/Gittyup#179] [Q&A]
Hi all, 
I need some suggestions. I'm working on a Gentoo ebuild for Gittyup and I have more or less completed it, however I'm facing a weird issue that I can't understand. When running Gittyup using my normal account, the menu bar is not shown, 
![image](https://user-images.githubusercontent.com/385249/170818578-f93feee0-98fd-4664-abd7-98117362d607.png)
but when running the same with sudo or using a completely new account, the bar is correctly shown. 
![image](https://user-images.githubusercontent.com/385249/170818552-e22ad7f4-5311-4ead-9dcb-edde9585bbb9.png)

The environment is linux/gentoo , KDE 5.24.5, Qt 5.15.4

I tried with strace, changing various configuration for the desktop environment, removing and recreating gittyup config files and so on. Probably I'm missing something horribly naive, but I can't figure it out. Any suggestion on something to try? Gitahead in the same setup works just fine.

Many thanks for any answer/hint/mild jokes :)

**Comment by cova-fe** [comment]
Interestingly enough, also the flatpak version has the same issue. So I guess it is something in my configuration that triggers this behavior, however it happens only with Gittyup, so I guess it could be worth investigating.

**Comment by cova-fe** [comment]
If someone is interested, the issue is triggered when KDE window decorations are configured to show the application menu in window bar. In this case (configuration shown below) the menus are not shown. This could be a gittyup issue or kde issue, maybe the first.
![image](https://user-images.githubusercontent.com/385249/171420968-0c21e2e1-a03a-4024-9e29-8c4aca7b7f23.png)

**Comment by r7l** [comment]
@cova-fe Is there any chance to get your Ebuild? I'd like to give this a try but there is no Ebuild for it.
